Header header
TrackedBoundingBox[] bounding_boxes
